src/main.cpp

```diff
@@ -271,6 +351,135 @@ void EMGetConfig() {
+  jsonResponse["mqtt"]["enable"] = true;                   // ToDO Echtzeitwerte
+  jsonResponse["mqtt"]["server"] = mqtt_server;
+  jsonResponse["mqtt"]["client_id"] = shelly_name;
+  jsonResponse["mqtt"]["user"] = nullptr;
+  jsonResponse["mqtt"]["ssl_ca"] = nullptr;
+  jsonResponse["mqtt"]["topic_prefix"] = mqtt_topic;
+  jsonResponse["mqtt"]["rpc_ntf"] = true;
+  jsonResponse["mqtt"]["status_ntf"] = true;
+  jsonResponse["mqtt"]["use_client_cert"] = false;
+  jsonResponse["mqtt"]["enable_rpc"] = true;
+  jsonResponse["mqtt"]["enable_control"] = true;
+  jsonResponse["sys"]["device"]["name"] = nullptr;
+  jsonResponse["sys"]["device"]["mac"] = shelly_mac;
+  jsonResponse["sys"]["device"]["fw_id"] = shelly_fw_id;
+  jsonResponse["sys"]["device"]["discoverable"] = true;
+  jsonResponse["sys"]["device"]["eco_mode"] = false;
+  jsonResponse["sys"]["location"]["tz"] = "Europe/Berlin";
+  jsonResponse["sys"]["location"]["lat"] = 51.342;
+  jsonResponse["sys"]["location"]["lon"] = 12.3819;
+  jsonResponse["sys"]["debug"]["level"] = 2;
+  jsonResponse["sys"]["debug"]["file_level"] = nullptr;
+  jsonResponse["sys"]["debug"]["mqtt"]["enable"] = false;
+  jsonResponse["sys"]["debug"]["websocket"]["enable"] = false;
+  jsonResponse["sys"]["debug"]["udp"]["addr"] = nullptr;
+  jsonResponse["sys"]["debug"]["rpc_udp"]["dst_addr"] = nullptr;
+  jsonResponse["sys"]["debug"]["rpc_udp"]["listen_port"] = nullptr;
+  jsonResponse["sys"]["debug"]["sntp"]["server"] = "time.google.com";
+  jsonResponse["sys"]["debug"]["cfg_rev"] = 13;
+  jsonResponse["wifi"]["ap"]["ssid"] = wifi_AP;
+  jsonResponse["wifi"]["ap"]["is_open"] = true;
+  jsonResponse["wifi"]["ap"]["enable"] = false;
+  jsonResponse["wifi"]["ap"]["range_extender"]["enable"] = false;
+  jsonResponse["wifi"]["sta"]["ssid"] = wifi_ssid;
+  jsonResponse["wifi"]["sta"]["is_open"] = false;
+  jsonResponse["wifi"]["sta"]["enable"] = true;
+  jsonResponse["wifi"]["sta"]["ipv4mode"] = "dhcp";
+  jsonResponse["wifi"]["sta"]["ip"] = nullptr;
+  jsonResponse["wifi"]["sta"]["netmask"] = nullptr;
+  jsonResponse["wifi"]["sta"]["gw"] = nullptr;
+  jsonResponse["wifi"]["sta"]["nameserver"] = nullptr;
+  jsonResponse["wifi"]["sta1"]["ssid"] = nullptr;
+  jsonResponse["wifi"]["sta1"]["is_open"] = true;
+  jsonResponse["wifi"]["sta1"]["enable"] = false;
+  jsonResponse["wifi"]["sta1"]["ipv4mode"] = "dhcp";
+  jsonResponse["wifi"]["sta1"]["ip"] = nullptr;
+  jsonResponse["wifi"]["sta1"]["netmask"] = nullptr;
+  jsonResponse["wifi"]["sta1"]["gw"] = nullptr;
+  jsonResponse["wifi"]["sta1"]["nameserver"] = nullptr;
+  jsonResponse["wifi"]["roam"]["rssi_thr"] = WiFi.RSSI();
+  jsonResponse["wifi"]["roam"]["interval"] = 60;
+  jsonResponse["ws"]["enable"] = false;
+  jsonResponse["ws"]["server"] = nullptr;
+  jsonResponse["ws"]["ssl_ca"] = "ca.pem";
+  serializeJson(jsonResponse,serJsonResponse);
+  DEBUG_SERIAL.println(serJsonResponse);
+}
+
+void ShellyGetStatus(){

```

Comment:
Die Daten in em:0 und emdata0 sollten besser die bereits existierenden Funktionen generieren.